### PR TITLE
Update What's New

### PR DESCRIPTION
**Pull request by ooeygui**

### CODE REVIEW

File: docs/WhatsNew.md [hunk 1]
```diff
@@ -1,5 +1,96 @@
 # What's New
-This page tracks changes to the ROS on Windows Documentation.
+This page tracks changes to the Azure Edge Robotics Documentation.
+
+## January 2021
+### ROS 2 Launch debugging in VSCode!
+Yes, you can now press F5 to launch a ROS2 node from VSCode and step through it.
+
+### Updated Depednencies
```
Comment (roycey) [inline docs/WhatsNew.md:8]: Typo

File: docs/WhatsNew.md [hunk 2]
```diff
@@ -1,5 +1,96 @@
...
+OPML with ROS Industrial
+
+### ROS Node bringup for UFactory XArm on ROS on Windows
+Brought up in colaboration with UFActory.
```
Comment (roycey) [inline docs/WhatsNew.md:12]: collaboration

**Review by roycey — CHANGES_REQUESTED** [review]
Except for a couple of typos. looks good to me

File: docs/WhatsNew.md (hunk 2 above)
Comment (ooeygui) [reply docs/WhatsNew.md:12]: Ack, I need to replace the keyboard on my laptop...

**Pull request merged**